packages/web/src/components/withdraw-usdc-modal/components/CoinflowWithdrawModal.tsx

```diff
@@ -46,9 +46,13 @@ export const CoinflowWithdrawModal = () => {
     onClose()
   }, [dispatch, onClose])
 
-  const handleSuccess = useCallback(
-    (params: string) => {
-      const transaction = parseTransactionFromSuccessParams(params)
+  const handleSuccess = useCallback<OnSuccessMethod>(
+    (args) => {
+      const transaction =
+        typeof args === 'object'
+          ? (args.hash ?? '')
```

Comment:
empty str default is ok?

Reply:
Yeah, the parse function also returns an empty string when it fails to parse the result.